question_4.rb

```diff
@@ -0,0 +1,44 @@
+# User input for mobile number & Check
+puts 'Enter mobile number to check: '
+mobile = gets.chomp
```

Comment:
Remove leading and trailing space to avoid interference with validation for each input in this file.